tests/Endpoints/ClientTest.php

```diff
@@ -141,6 +141,14 @@ public function testGetIndex()
         $this->assertNull($index->getPrimaryKey());
     }
 
+    public function testCreateOrGetIndex()
+    {
+        $index = $this->client->getOrCreateIndex('index');
+        $this->assertInstanceOf(Index::class, $index);
+        $this->assertSame('index', $index->getUid());
+        $this->assertNull($index->getPrimaryKey());
+    }
```

Comment:
Can you check the method works even if the index already exists?
You can be inspired by what's been done here in [meilisearch-python](https://github.com/meilisearch/meilisearch-python):

https://github.com/meilisearch/meilisearch-python/blob/d6153c03ee87f09100af826fb33930cf115faab1/meilisearch/tests/index/test_index.py#L60-L84

Reply:
@curquiza added a new method `testGetOrCreateWithIndexAlreadyExists`